File: Form42.vb
Imports System.ComponentModel
Imports System.Windows.Forms

Public Class Form42
    Dim form1 As Form29_Simple_Drop_down_List = Nothing
    Dim form2 As Form30_Create_Dynamic_Drop_down_List = Nothing
    Private Sub RadioButton5_CheckedChanged(sender As Object, e As EventArgs) Handles RB_No.CheckedChanged
        If RB_No.Checked = True Then
            CGB.Enabled = False
            RB_Simple.Enabled = False
            RB_Dynamic.Enabled = False
        End If
    End Sub

    Private Sub RB_Yes_CheckedChanged(sender As Object, e As EventArgs) Handles RB_Yes.CheckedChanged
        If RB_Yes.Checked = True Then
            CGB.Enabled = True
            RB_Simple.Enabled = True
            RB_Dynamic.Enabled = True

        End If
    End Sub

    Private Sub Form42_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub Btn_Cancel_Click(sender As Object, e As EventArgs) Handles Btn_Cancel.Click
        Me.Close()
    End Sub

    Private Sub Btn_OK_Click(sender As Object, e As EventArgs) Handles Btn_OK.Click
        If RB_Simple.Checked = True And RB_Simple.Enabled = True Then
            form1 = New Form29_Simple_Drop_down_List()
            Me.Hide()
            form1.Show()
        ElseIf RB_Dynamic.Checked = True And RB_Dynamic.Enabled = True Then
            form2 = New Form30_Create_Dynamic_Drop_down_List()
            Me.Hide()
            form2.Show()
        Else
            Me.Dispose()
        End If
        Me.Close()

    End Sub

    Private Sub CheckBox1_CheckedChanged(sender As Object, e As EventArgs) Handles CheckBox1.CheckedChanged
        If CheckBox1.Checked = True Then
            sessionflag2 = False
        Else
            sessionflag2 = True
        End If
    End Sub

    Private Sub Form42_Closing(sender As Object, e As CancelEventArgs) Handles Me.Closing
        form_flag = False
    End Sub

    Private Sub Form42_Disposed(sender As Object, e As EventArgs) Handles Me.Disposed
        form_flag = False
    End Sub

    Private Sub RB_Simple_CheckedChanged(sender As Object, e As EventArgs) Handles RB_Simple.CheckedChanged

    End Sub
End Class

File: Form42.cs
using System;
using System.ComponentModel;

namespace VSTO_Addins
{

    public partial class Form42
    {
        private Form29_Simple_Drop_down_List form1 = null;
        private Form30_Create_Dynamic_Drop_down_List form2 = null;

        public Form42()
        {
            InitializeComponent();
        }
        private void RadioButton5_CheckedChanged(object sender, EventArgs e)
        {
            if (RB_No.Checked == true)
            {
                CGB.Enabled = false;
                RB_Simple.Enabled = false;
                RB_Dynamic.Enabled = false;
            }
        }

        private void RB_Yes_CheckedChanged(object sender, EventArgs e)
        {
            if (RB_Yes.Checked == true)
            {
                CGB.Enabled = true;
                RB_Simple.Enabled = true;
                RB_Dynamic.Enabled = true;

            }
        }

        private void Form42_Load(object sender, EventArgs e)
        {

        }

        private void Btn_Cancel_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void Btn_OK_Click(object sender, EventArgs e)
        {
            if (RB_Simple.Checked == true & RB_Simple.Enabled == true)
            {
                form1 = new Form29_Simple_Drop_down_List();
                Hide();
                form1.Show();
            }
            else if (RB_Dynamic.Checked == true & RB_Dynamic.Enabled == true)
            {
                form2 = new Form30_Create_Dynamic_Drop_down_List();
                Hide();
                form2.Show();
            }
            else
            {
                Dispose();
            }
            Close();

        }

        private void CheckBox1_CheckedChanged(object sender, EventArgs e)
        {
            if (CheckBox1.Checked == true)
            {
                GlobalModule.sessionflag2 = false;
            }
            else
            {
                GlobalModule.sessionflag2 = true;
            }
        }

        private void Form42_Closing(object sender, CancelEventArgs e)
        {
            GlobalModule.form_flag = false;
        }

        private void Form42_Disposed(object sender, EventArgs e)
        {
            GlobalModule.form_flag = false;
        }

        private void RB_Simple_CheckedChanged(object sender, EventArgs e)
        {

        }
    }
}